app-text/zathura/zathura-0.5.11.ebuild

```
@@ -48,20 +47,18 @@ BDEPEND="
 	>=sys-devel/gettext-0.19.8
 	virtual/pkgconfig
 "
+
 src_configure() {
 	local emesonargs=(
 		-Dconvert-icon=disabled
-		-Dmanpages=$(usex man enabled disabled)
-		-Dseccomp=$(usex seccomp enabled disabled)
-		-Dsynctex=$(usex synctex enabled disabled)
-		)
+		-Dlandlock=enabled
+		$(meson_feature man manpages)
+		$(meson_feature seccomp)
+		$(meson_feature synctex)
```

```suggestion
		$(meson_feature synctex)
		$(meson_feature test tests)
```

https://github.com/pwmt/zathura/blob/9c9380e57d83eb3638348710965fc896f7f71709/meson_options.txt#L21